automation/actions/cacheLPTTotalSupply/index.ts

```diff
@@ -0,0 +1,40 @@
+import {ActionFn, Context, Event} from '@tenderly/actions';
+import {ethers} from 'ethers';
+import {waitForTx, waitToRelayTxsToL2} from '../../../utils/arbitrum';
+import {getArbParams, getCache} from './helpers';
+import {ARBITRUM_NETWORK} from '../../../deploy/constants';
+
+export const triggerCache: ActionFn = async (
+    context: Context,
+    _event: Event,
+) => {
+  const infuraKeyL1 = await context.secrets.get('INFURA_KEY_ETH');
+  const infuraKeyL2 = await context.secrets.get('INFURA_KEY_ARB');
```

Comment:
is it necessary for the rpc endpoint to be infura? can it be alchemy / another provider? If yes, can we rename it?

Reply:
I noticed that the comments to run the Foundry tests in the protocol repo were recommending Alchemy